src/aosm/setup.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+from codecs import open
+from setuptools import find_packages, setup
+
+try:
+    from azure_bdist_wheel import cmdclass
+except ImportError:
+    from distutils import log as logger
+
+    logger.warn("Wheel is not available, disabling bdist_wheel hook")
+
+# Confirm this is the right version number you want and it matches your
+# HISTORY.rst entry.
+VERSION = "1.0.0b1"
+
+# The full list of classifiers is available at
+# https://pypi.python.org/pypi?%3Aaction=list_classifiers
+CLASSIFIERS = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "Intended Audience :: System Administrators",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "License :: OSI Approved :: MIT License",
+]
+
+DEPENDENCIES = ["oras~=0.1.19", "azure-storage-blob>=12.15.0", "jinja2>=3.1.2"]
```

Comment:
Just out of curiosity, can the dependency `azure-storage-blob` be included by the dependency `azure-multiapi-storage` and `azure-storage-common` in the azure-cli-core? [code link](https://github.com/Azure/azure-cli/blob/be065f1cab2be4549c9d5d4e57cd0e69c904f441/src/azure-cli/setup.py#L128-L129)

Reply:
`azure-storage-blob` depends on `azure-storage-common`  (not vice versa).

https://pypi.org/project/azure-multiapi-storage/ suggests `azure-multiapi-storage` might cover `azure-storage-blob`, but I don't know how to use it, and would rather not make an important dependency change at this late stage. I've raised work item https://dev.azure.com/msazuredev/AzureForOperators/_workitems/edit/992831 to track.